src/ODEmodel.jl

```diff
@@ -69,7 +69,7 @@ function predict(p, g_0::Real, t, nG1::Integer, nG2::Integer, nD1, nD2)
     end
 
     v = [ones(nG1) * p[5] * g_0 / nG1; ones(nG2) * (1.0 - p[5]) * g_0 / nG2; D1; D2]
-    A = ODEjac(p, t[2], nG1, nG2, nD1, nD2)
```

Comment:
How are you trying to adjust predict? With this change, it will only give you data with 30 min timesteps, which seems to *not* be what you want when you use it elsewhere.

Reply:
I will change this one, too. What I wanted to do was using the last data point from drug1 to be used as an initial condition in drug2.